packages/core/tests/storage.test.ts

```diff
@@ -37,37 +37,9 @@ describe("core/storage", () => {
           };
         },
         { overwriteRoutes: false }
-      )
-      .get("https://fake.broker.dev/storagerequest/fakeId", () => {
```

Comment:
no longer mocking this, and just testing it live... as we don't require a token for this api